Traceback (most recent call last):
  File "/Users/rudra_sarkar/Documents/PhD IIT Madras/Intro to Deep Learning M Khapra DA6401/DA6401 Assignment/da6401_assignment1/train.py", line 24, in <module>
    wandb.config(configuration_script)
TypeError: 'Config' object is not callable
